.github/workflows/eval.yml

```
@@ -251,6 +266,20 @@ jobs:
               /repos/"$REPOSITORY"/issues/"$NUMBER"/labels \
               -f "labels[]=$toAdd"
           done < <(comm -13 before after)
+
+          # maintainers.json contains GitHub IDs. Look up handles to request reviews from.
+          # There appears to be no API to request reviews based on GitHub IDs
+          jq -r 'keys[]' comparison/maintainers.json \
+            | while read -r id; do gh api /user/"$id"; done \
+            | jq -s '{ reviewers: map(.login) }' \
+            > reviewers.json
```

```suggestion
          jq -r 'keys[]' comparison/maintainers.json \
            | head -n15 \
            | while read -r id; do gh api /user/"$id"; done \
            | jq -s '{ reviewers: map(.login) }' \
            > reviewers.json
```
GitHub apps are limited to 15 reviewer requests anyway, and this way treewide changes are handled better. Adding a `head -n15` here should work, but I did not test.

---

Sounds like a good idea. Should be the list sorted, so it's always the same 15 reviewers?

---

I think there is also a limit with posting maintainers in comments.